source/pages/Transactions/Transaction.tsx

```diff
@@ -42,16 +24,29 @@ interface ITransaction {
 const Transaction: React.FC<ITransaction> = ({ type }) => {
   const { host, ...transaction } = useQueryData();
   const navigate = useNavigate();
+  const {
+    wallet: { txs },
+  } = getController();
 
   const [fee, setFee] = useState<number>();
 
+  const {
+    vault: { isBitcoinBased, activeNetwork },
+  } = useSelector((state: RootState) => state);
```

Comment:
Writing the selector like this will trigger an UI update on any modification at the store

Reply:
[changed](https://github.com/syscoin/pali-wallet/pull/261#discussion_r966310781)